components/NavBar/index.tsx

```diff
@@ -43,15 +47,23 @@ const NavBar: FunctionComponent = () => {
                         </StyledNavBarItemLink>
                         <SiteSearch />
                     </StyledNavBarItemsContainer>
-                    <div>                
+                    <StyledButtonsWrapper>                
                         <Button 
                             href="https://sourcegraph.com/search" 
                             className="outline-primary"
                             target="_blank"
                             rel="noreferrer">
                             Search on Sourcegraph
-                        </Button>                     
-                    </div>
+                        </Button>
+                        <Button
+                            className="flex"
+                            target="_blank"
+                            rel="noreferrer"
+                            onClick={() => theme.toggleTheme()}>
+                            <LighteningBoltOutlineIcon />
+                            Dark Mode
```

Comment:
Can we say something more generic than `Dark Mode` since it also turns on "Light Mode"? Here is dark mode with link displaying "dark mode"

<img width="377" alt="Screen Shot 2021-11-11 at 11 12 41 PM" src="https://user-images.githubusercontent.com/5977693/141408203-da8091a0-5819-451c-bc87-aa84f0830489.png">

I like how Tania Rascia uses `Dark` / `Light` on her site (also toggles on/off with spacebar): https://www.taniarascia.com/

<img width="218" alt="Screen Shot 2021-11-11 at 11 20 57 PM" src="https://user-images.githubusercontent.com/5977693/141408954-9768fd14-49c9-4bc9-86e7-40948fe1ea23.png"> 
<img width="220" alt="Screen Shot 2021-11-11 at 11 21 17 PM" src="https://user-images.githubusercontent.com/5977693/141408983-7f074a7c-35b3-4787-a468-6bf5cb12e2a8.png">

Microsoft Docs have a Theme menu in the footer: https://docs.microsoft.com/en-us/windows-hardware/drivers/

<img width="616" alt="Screen Shot 2021-11-11 at 11 20 38 PM" src="https://user-images.githubusercontent.com/5977693/141408935-0e3d8325-6ad1-4d0f-badf-843f7b55120c.png">

Maybe a moon vs sun toggle instead of the lightning bolt? Not sure what would really resonate.